src/main/java/com/revature/repositories/CarRepository.java

```diff
@@ -24,6 +24,16 @@
 	 * @return Check {@link com.revature.services.impl.UserServiceImpl}
 	 */
 	
-	@Query("select c from Car c where c.user.userId = ?1")
+	//@Query("select c from Car c where c.user.userId = ?1")
```

Comment:
can this be removed?

Reply:
I think this was from my pull request from earlier. We're working on a bug fix so we can remove that when we push later tonight.

Reply:
removed